### PR TITLE
Fogtesting

### PR DESCRIPTION
**Pull request by Manghi**
Added unit tests for recent fog improvements. 
I didn't test with fog radius > 64 (maybe you can do this one)?

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -1379,6 +1384,312 @@ mod universe_tests {
...
+            vec![0x01ffffffffffffff],
+            vec![0x01ffffffffffffff],
+            vec![0x01ffffffffffffff],
+            vec![0x01ffffffffffffff],
+            vec![0x83ffffffffffffff],
+        ];
+        uni.generate_fog_circle_bitmap(4);
+        assert_eq!(fog_radius_of_four, uni.fog_circle);
+
+        let fog_radius_of_hunned = vec![
```
Comment (AaronM04) [inline src/lib.rs]: Impressive, but damn, that's taking up a lot of space in the file... How about a radius 35 circle instead? That is much smaller but still tests multi-u64 rows.

**Review by AaronM04 — CHANGES_REQUESTED** [review]
Please change the gigantic unit test. Otherwise looks great! 👍

File: src/lib.rs (hunk 1 above)
Comment (Manghi) [reply src/lib.rs]: Don't want to make it 💯 ? Sure I can trim this down. :)

**Comment by Manghi** [comment]
@AaronM04 is 35 still too big?

**Review by AaronM04 — APPROVED** [review]
LGTM!

**Pull request merged**